debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectionFactory.java

```diff
@@ -5,32 +5,33 @@
  */
 package io.debezium.connector.oracle;
 
-import static io.debezium.connector.oracle.OracleConnectorConfig.CONNECTOR_ADAPTER;
-
 import java.sql.Connection;
-import java.sql.DriverManager;
 import java.sql.SQLException;
 
 import io.debezium.connector.oracle.OracleConnectorConfig.ConnectorAdapter;
 import io.debezium.jdbc.JdbcConfiguration;
+import io.debezium.jdbc.JdbcConnection;
 import io.debezium.jdbc.JdbcConnection.ConnectionFactory;
 
 public class OracleConnectionFactory implements ConnectionFactory {
```

Comment:
BTW, as I think about that, do we need the indirect construction via `ConnectionFactory`?  Check how it is done with `SqlServerConnection`. Maybe it might make sense to simplify the connection creation in this PR to make it more understandable.

Reply:
We can probably do that if I can understand the issue with the `USE_FETCH_SIZE_WITH_LONG_COLUMN` issue where it isn't part of the JdbcConfiguration and I have to force apply it when its not provided.

Reply:
I removed this class entirely, I was able to leverage creating the factory similar to how SQL Server did.